src/SDKs/Reservations/Reservations.Tests/Reservations.Tests.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Reservations.Tests</PackageId>
+    <Description>Reservations.Tests Class Library</Description>
+    <Authors>Microsoft Corporation</Authors>
+    <AssemblyName>Reservations.Tests</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+
+  <!--
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.ResourceManager" Version="1.1.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Test.HttpRecorder" Version="1.8.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0-preview-20170810-02" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.9" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.7.2" />
+    <PackageReference Include="xunit" Version="2.2.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.0-beta3-build3705" />
+  </ItemGroup>
+  -->
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Azure.Management.Reservations\Microsoft.Azure.Management.Reservations.csproj" />
```

Comment:
@corquiri add reference to latest version of resource manager nuget package and record test using the latest version of resource manager

Reply:
![capture](https://user-images.githubusercontent.com/31109331/32469916-b92e4d6a-c309-11e7-921d-f0e7f5b033a1.PNG)

RM reference is already in Nuget dependency of Test project as shown